resources/scss/modules/_buttons.scss

```diff
@@ -4,29 +4,20 @@
     line-height: normal;
 
     &:not(.flat) {
-        border: 2px solid $color-accent-main-base;
-        color: $color-accent-main-base;
+        background-color: $color-accent-main-base;
+        color: $color-accent-main-text;
         border-radius: 24px;
         padding: $spacing-micro $spacing-small;
-
         text-align: center;
-
-        transition: background-color .1s ease-in-out;
+        transition: .3s ease-in-out;
     }
 
     &:not(.flat):hover {
-        background-color: $color-accent-main-base;
-        color: $color-accent-main-text;
+        box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```

Comment:
I find a subtler shadow works better in this case:

```css
box-shadow: 0 4px 6px rgba(0,0,0,0.12), 0 2px 4px rgba(0,0,0,0.1);
```

This is in part because these buttons can be on different backgrounds, on white it looks like this, coupled with making the background color lighter:

![Screenshot from 2020-02-12 09-46-42](https://user-images.githubusercontent.com/12694995/74351343-9f2c8580-4d7c-11ea-9017-54b385eea3a0.png)

Also two things:

- The box shadow should use a variable or mixin, so we can reuse it on all hovered and clickable elements, not just this button.
- I think the hover effect would be clearer if the background-color became lighter. The lighten function should do the trick I tested with `background-color: #46d1f1;` for the blue button and it reads well to me.